changelog.dd

```diff
@@ -358,6 +362,56 @@ assert(a[0] == 42);
 -------
 )
 
+$(LI $(LNAME2 inner-class, `std.traits.isInnerClass` was added to identify
+    Java-style nested classes with an accessible `outer` pointer)
```

Comment:
Just call them nested classes, not "Java-style" nested classes. People might think there's a difference to regular nested-classes, or they might think these have special support for interfacing with Java.